tsconfig.json

```diff
@@ -1,17 +1,21 @@
 {
     "compilerOptions": {
-        "target": "es6",
-        "module": "commonjs",
+        "target": "ES2017",
+        "module": "CommonJS",
         "esModuleInterop": true,
         "strict": true,
         "incremental": true,
         "sourceMap": true,
         "resolveJsonModule": true,
         "allowJs": false,
         "declaration": true,
+        "declarationMap": true,
         "noUnusedParameters": true,
-        "noUnusedLocals": true
+        "noUnusedLocals": true,
+        "moduleResolution": "node",
+        "inlineSources": true,
+        "importHelpers": true
```

Comment:
According to the docs, this requries the "tslib" at runtime, which we don't explicitly bundle. I don't think we need this on